src/backend/vulkan/src/lib.rs

```diff
@@ -57,7 +57,19 @@ mod window;
 
 // CStr's cannot be constant yet, until const fn lands we need to use a lazy_static
 lazy_static! {
-    static ref LAYERS: Vec<&'static CStr> = vec![#[cfg(debug_assertions)] CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_standard_validation\0").expect("Wrong extension string")];
+    static ref LAYERS: Vec<&'static CStr> = if cfg!(all(target_os = "android", debug_assertions)) {
+        vec![
+            CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_core_validation\0").expect("Wrong extension string"),
+            CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_object_tracker\0").expect("Wrong extension string"),
+            CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_parameter_validation\0").expect("Wrong extension string"),
+            CStr::from_bytes_with_nul(b"VK_LAYER_GOOGLE_threading\0").expect("Wrong extension string"),
+            CStr::from_bytes_with_nul(b"VK_LAYER_GOOGLE_unique_objects\0").expect("Wrong extension string"),
+        ]
+    } else if cfg!(debug_assertions) {
+        vec![CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_standard_validation\0").expect("Wrong extension string")]
```

Comment:
let's just `unwrap` or `from_bytes_with_nul_unchecked`. There is not much use from those `expect` messages

Reply:
Replaced with unwrap